### PR TITLE
Update change-site-address.md

### PR DESCRIPTION
**Pull request by WayneEwington**
Document that Site Rename is unsupported if the publishing feature been activated or has been previously activated.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/change-site-address.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ As a global or SharePoint admin in your organization, you can change the URL for
 - Communication sites
 - Classic team sites
 
+>[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:39]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/change-site-address.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ As a global or SharePoint admin in your organization, you can change the URL for
...
+>If the Publishing Feature has been activated or has been previously activated in the site, then performing a Site Rename is unsupported.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:40]:
```suggestion
> If the Publishing feature has been activated or has been previously activated in the site, then performing a Site Rename is unsupported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**